Documentation/cassandra.md

```diff
@@ -1,4 +1,10 @@
-# Quickstart
+---
+title: Cassandra
+weight: 5
```

Comment:
I think I got it! 
I gave each file unique numbers, but they're running out (I just used the last one from the 30-39 range for the `cassandra-cluster-crd/md`. I moved all the ceph docs at the top since they are the most stable storage provider.
We should refactor the doc structure soon as per #1802.

Reply:
i'm still seeing this one at weight 5, which would conflict with cockroach.  did you push all the changes you made?

Reply:
I must have been so tired yesterday that I forgot to push :disappointed: 
Now everything should be ok ! :grin: